scripts/ray_autoscale.yaml

```diff
@@ -40,8 +40,10 @@ auth:
 # http://boto3.readthedocs.io/en/latest/reference/services/ec2.html#EC2.ServiceResource.create_instances
 head_node:
     InstanceType: c4.4xlarge
+    # 04b53bf506e95394f
     #ImageId: ami-060b3311dd2b675ee # Flow AMI (Ubuntu)
-    ImageId: ami-0067051ba5c8f0b64 # Flow AMI (Ubuntu)
+    # ImageId: ami-0067051ba5c8f0b64 # Flow AMI (Ubuntu)
+    ImageId: ami-09ee73f99ee3510b0 # kathy AMI (Ubuntu)
```

Comment:
Hey Kathy, just as a check, have you made sure there are no SSH keys or any personal information on this AMI?

Reply:
good point. it's private right now so I'll keep the PR up. should i just remove everything in ~/.ssh?

Reply:
Yes, definitely.